security/encrypt-cluster-pod-traffic.md

```diff
@@ -36,7 +36,92 @@ This how-to guide uses the following {{site.prodname}} features:
 
 1. Install WireGuard on cluster nodes using {% include open-new-window.html text='instructions for your operating system' url='https://www.wireguard.com/install/' %}. Note that you may need to reboot your nodes after installing WireGuard to make the kernel modules available on your system.
```

Comment:
should we not just tell user to reboot after installing? Like this:

1. Install WireGuard on cluster nodes using <>
1. Reboot nodes.

Reply:
Reboot is not mandatory for all platforms e.g. EKS, Ubuntu in managed k8s case. It's however required for OpenShift.